php_phongo.c

```diff
@@ -245,8 +245,6 @@ void phongo_cursor_init(zval *return_value, mongoc_cursor_t *cursor, const bson_
 	intern->cursor = cursor;
 	intern->server_id = mongoc_cursor_get_hint(cursor);
 	intern->client = client;
-	intern->is_command_cursor = is_command_cursor;
```

Comment:
I suggest removing this because, after simplifying our iteration handlers, it would only be used in the Cursor's debug handler.